README.md

```diff
@@ -5,3 +5,40 @@ LiClipseText is an editor which enables Eclipse to be used as a general-purpose
 It's the Open Source (EPL-based) counterpart of LiClipse (http://www.lilcipse.com).
 
 See: www.liclipse.com/text for details
+
+Supported languages (taken from [plugin.xml](https://github.com/fabioz/LiClipseText/blob/master/plugins/org.brainwy.liclipsetext.editor/plugin.xml#L163))
```

Comment:
Humm, I think you could get the list from https://marketplace.eclipse.org/content/liclipsetext -- which I just updated (where it's better formatted so that it's not all lowercase).